Source/HelixToolkit.Wpf.Shared/Geometry/Plane3D.cs

```diff
@@ -19,12 +19,12 @@ public class Plane3D
         /// <summary>
         /// The normal.
         /// </summary>
-        private Vector3D normal;
+        private Vector3D _normal;
```

Comment:
Avoid adding the prefix `_` for private variables.

Reply:
It's a project wide change, so I would suggest to keep current convention is better for this CR.